partner_company_group/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -9,4 +9,12 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.rename_fields(env, field_renames)
+    if openupgrade.column_exists(env.cr, "res_partner", "group_id"):
+        if not openupgrade.column_exists(env.cr, "res_partner", "company_group_id"):
+            openupgrade.rename_fields(env, field_renames)
+        else:
+            openupgrade.logged_query(
+                env.cr, """
+                UPDATE res_partner
+                SET company_group_id = group_id
```

Comment:
The target model is the same?

Reply:
yes, both are res.partner

Reply:
I could add a query to check both field target same model, but that would make dirty the code

Reply:
OK, just want to make sure of that.